ctt/process_dependencies.py

```diff
@@ -420,12 +424,20 @@ def process_images(
 
     executor = concurrent.futures.ThreadPoolExecutor(max_workers=16)
 
+    reftype_filter = None
+    if skip_labels and skip_labels(dso.labels.ExtraComponentReferencesLabel.name):
+        def filter_extra_component_refs(reftype: cnudie.iter.NodeReferenceType) -> bool:
+            return reftype == cnudie.iter.NodeReferenceType.EXTRA_COMPONENT_REFS_LABEL
```

Comment:
nit: use `is` for comparison w/ enum-values?

did you use `==` to gracefully handle cases where we encounter a str-value we do not know as of our enum?

Reply:
No good point, I did not do this on purpose. The `reftype` here stems from `cnudie.iter`, so we can assume that it's a value of our enum or at least fail if it's not 👍🏼